clients/unshaded/src/main/java/tachyon/client/block/LocalBlockOutStream.java

```diff
@@ -37,6 +38,7 @@
  * thread and are not thread safe.
  */
 public class LocalBlockOutStream extends BufferedBlockOutStream {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
```

Comment:
As a convention, we have a blank line between static variables and member variables.